app/lib/RouteResolver.js

```diff
@@ -70,13 +82,15 @@ function RouteResolver(app) {
 RouteResolver.prototype.register = function register(method, path, response) {
   const route = { method, path, response };
 
+  route.pathname = parse(route.path, true).pathname;
+  const matchKeys = [];
+  route.pathRegExp = pathToRegExp(route.pathname, matchKeys);
+  route.matchKeys = matchKeys;
```

Comment:
It's not very clear that pathToRegExp is mutating matchKeys. A comment here would be helpful.

Reply:
Will add comment.